_includes/homepage_content_tr.html

```diff
@@ -145,7 +145,7 @@ <h1 class="intro text-monospace display-1" style="margin-left: -.3rem;">{{ item.
 
               </div>
               <div class="col-md-12 order-md-1 text-md-left homepage-hero">
-                <img class="img-fluid shadow-lg" alt="bisq" style="width: 100%; display: block;" src="/{{ page.lang }}/images/bisq_screen0.png">
+                <img class="img-fluid shadow-lg" alt="Offer Book view inside Bisq" style="width: 100%; display: block;" src="/images/bisq_screen0.svg">
```

Comment:
```suggestion
                <img class="img-fluid shadow-lg" alt="Offer Book view inside Bisq" style="width: 100%; display: block;" src="/{{ page.lang }}/images/bisq_screen0.png">
```

Why was image source changed here?